src/extension/features/budget/link-to-inflows/index.js

```diff
@@ -0,0 +1,58 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import {
+  getSelectedMonth,
+  isCurrentRouteBudgetPage,
+  transitionTo,
+} from 'toolkit/extension/utils/ynab';
+
+/**
+ * Adds a click handler to the "TOTAL INFLOW" inspector area.
+ */
+export class LinkToInflows extends Feature {
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage();
+  }
+
+  onRouteChanged() {
+    if (this.shouldInvoke()) {
+      this.invoke();
+    }
+  }
+
+  /**
+   * Add the wrapper element to the TOTAL INFLOWS inspector area.
+   */
+  invoke() {
+    $('.budget-inspector')
+      .find(`h3:contains("${Ember.I18n.t('inspector.totalIncome')}")`)
+      .next()
+      .andSelf()
+      .wrapAll('<span class="toolkit-total-inflows" />');
+
+    $('.toolkit-total-inflows').click(this.onClick);
+  }
+
+  /**
+   * Handle the click on TOTAL INFLOWS. Set the search field and navigate
+   * to the accounts route.
+   */
+  onClick() {
+    const month = getSelectedMonth();
+    const controller = controllerLookup('accounts');
+
+    if (controller.filters) {
+      controller.filters.resetFilters();
+    }
+
+    controller.set(
+      'searchText',
+      `${Ember.I18n.t('search.income')}: ${month.format('MMMM YYYY')}`
```

Comment:
same here I reckon but maybe their search does wonkier things, I really haven't a clue if they localize that stuff it would kinda make sense that they do

Reply:
Yeah, I wasn't sure if hardcoding "Income" in english would break the search in other languages or not. From debugging what happens when you click on their suggested "income in month" search it seems like they're using the localized string... which seems a little strange. I tried to get ynab to render in another language so I could test, but then I gave up. 🙃  Do you know of an easy way to test in a non-english language?

Reply:
the only way I know of is to turn on the localization from toolkit - I actually don't think they have any other languages which leads me to think you probably wanna just hard code `Income` 😅

Reply:
Oh! I assumed YNAB supported other languages since I found a big ol' map of strings - 
<img width="634" alt="screenshot_3_3_19__1_22_pm" src="https://user-images.githubusercontent.com/1299305/53699561-a51d9680-3db7-11e9-8846-c2af1ca4f0e5.png">

And they seem to be using it - 
<img width="685" alt="screenshot_3_3_19__1_26_pm" src="https://user-images.githubusercontent.com/1299305/53699585-f2016d00-3db7-11e9-9bda-246236a9c31f.png">

But maybe that's just how they manage strings and is not actually used for localized?

I agree it makes sense to hardcode "income" here... that's guaranteed to work at least. ¯\_(ツ)_/¯